src/components/elements/slideshow.tsx

```diff
@@ -0,0 +1,63 @@
+'use client';
+import React, { useRef } from 'react';
+import Slider, { CustomArrowProps } from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import { ArrowLeftIcon, ArrowRightIcon } from '@heroicons/react/16/solid';
+
+export type SlideshowProps = {
+  children: React.ReactNode;
```

Comment:
Change to `React.JSX.Element`

Reply:
One thing I normally do is `type Props = HTMLAttributes<HTMLDivElement> & {}` That inherits the `children` prop.

Reply:
That would also satisfy my other comment on the actual component.